dvc/objects/db.py

```diff
@@ -295,81 +303,48 @@ def _list_hashes_traverse(
-        ) as pbar:
-            from funcy import collecting
-
-            with ThreadPoolExecutor(
-                max_workers=jobs or self.fs.jobs
-            ) as executor:
-                list_hashes = collecting(pbar.wrap_fn(self._list_hashes))
-                in_remote = executor.map(list_hashes, traverse_prefixes)
-                yield from itertools.chain.from_iterable(in_remote)
-
-    def all(self, jobs=None, name=None):
+
+        list_hashes = collecting(self._list_hashes)
+        with ThreadPoolExecutor(max_workers=jobs or self.fs.jobs) as executor:
+            in_remote = executor.map(list_hashes, traverse_prefixes)
+            yield from itertools.chain.from_iterable(in_remote)
+
+    def all(self, jobs=None):
         """Iterate over all hashes in this fs.
 
         Hashes will be fetched in parallel threads according to prefix
         (except for small remotes) and a progress bar will be displayed.
         """
-        logger.debug(
-            "Fetching all hashes from '{}'".format(
-                name if name else "remote cache"
-            )
-        )
-
         if not self.fs.CAN_TRAVERSE:
             return self._list_hashes()
 
-        remote_size, remote_hashes = self._estimate_remote_size(name=name)
+        remote_size, remote_hashes = self._estimate_remote_size()
         return self._list_hashes_traverse(
-            remote_size, remote_hashes, jobs=jobs, name=name
+            remote_size, remote_hashes, jobs=jobs
         )
 
     def _remove_unpacked_dir(self, hash_):
         pass
 
-    def list_hashes_exists(self, hashes, jobs=None, name=None):
+    def list_hashes_exists(self, hashes, jobs=None):
         """Return list of the specified hashes which exist in this fs.
         Hashes will be queried individually.
         """
         logger.debug(f"Querying {len(hashes)} hashes via object_exists")
-        with Tqdm(
-            desc="Querying "
-            + ("cache in " + name if name else "remote cache"),
-            total=len(hashes),
-            unit="file",
-        ) as pbar:
-
-            def exists_with_progress(fs_path):
-                ret = self.fs.exists(fs_path)
-                pbar.update_msg(fs_path)
-                return ret
-
-            with ThreadPoolExecutor(
-                max_workers=jobs or self.fs.jobs
-            ) as executor:
-                fs_paths = map(self.hash_to_path, hashes)
-                in_remote = executor.map(exists_with_progress, fs_paths)
-                ret = list(itertools.compress(hashes, in_remote))
-                return ret
-
-    def hashes_exist(self, hashes, jobs=None, name=None):
+        with ThreadPoolExecutor(max_workers=jobs or self.fs.jobs) as executor:
+            fs_paths = map(self.hash_to_path, hashes)
+            in_remote = executor.map(self.fs.exists, fs_paths)
+            yield from itertools.compress(hashes, in_remote)
+
+    def hashes_exist(self, hashes, jobs=None, progress=noop):
```

Comment:
I am trying not to touch this at the current stage, felt like removing the progress bar should be done first.